src/cascadia/TerminalApp/dll/TerminalApp.vcxproj

```diff
@@ -75,6 +75,28 @@
     </ProjectReference>
   </ItemGroup>
 
+
+  <ItemGroup>
+    <!-- Manually add a reference to TerminalControl here. We need this so
+    MDMERGE will know where the TermControl types are defined. However, we need
+    to do it exactly like this so the packaging project won't roll up
+    TermControl's .xbf's from both the TermControl project and this one. -->
+<!--     <Reference Include="Microsoft.Terminal.Control">
+      <HintPath>$(OpenConsoleCommonOutDir)TerminalControl\Microsoft.Terminal.Control.winmd</HintPath>
+      <IsWinMDFile>true</IsWinMDFile>
+      <Private>false</Private>
+      <CopyLocalSatelliteAssemblies>false</CopyLocalSatelliteAssemblies>
+    </Reference> -->
+

```

Comment:
This comment spells doom and gloom if we _don't_ do this thing, but then it's commented out. Why do we NOT need it?

Reply:
I'm gonna be real with you - no idea why this works now. maybe it thinks the .xbfs belong to TerminalControlLib now, not TerminalControl.dll?